build/helper/metadata_add_all.py

```diff
@@ -300,24 +309,25 @@ def test_add_all_metadata_simple():
                     'library_method_call_snippet': 'self._vi',
                 },
                 {
-                    'ctypes_type': 'ViString',
+                    'ctypes_type': 'ViChar',
                     'ctypes_variable_name': 'channel_name_ctype',
-                    'ctypes_type_library_call': 'ViString',
+                    'ctypes_type_library_call': 'ctypes.POINTER(ViChar)',
                     'direction': 'in',
                     'documentation': {
                         'description': 'The channel to call this on.'
                     },
                     'is_repeated_capability': True,
                     'enum': None,
-                    'python_type': 'str',
-                    'is_buffer': False,
+                    'python_type': 'int',
```

Comment:
shouldn't this be str?

Reply:
No. ViChar is int. See #422 where I handle it in documentation.

Reply:
I thought that the 'python_type' metadata value refers to the type of the parameter passed in to the Python API. But now that I think of it, that wasn't the case with buffers before.

How would you define this metadata value?

Reply:
We've settled this offline, and came up with the following definitions for metadata, whose names can be improved but would happen in a separate PR.

Python type metadata:

* *type*: “Simple” type of a parameter in the driver’s C API functions.
* *python_type*: “Simple” type of a parameter for the Python API (methods of Session). Can be int, float, boolean, enum (but not lists nor str)
* *is_buffer*: Is it a list or str
* *enum*: Which enum to use in the Python API.
* *direction*: Is it an input or output parameter.